index.bs

```diff
@@ -554,26 +554,26 @@ spec: XHR; urlPrefix: https://xhr.spec.whatwg.org/
   # Impact on Web Platform # {#impact}
 
   Content inside a suborigin namespace is restricted in the same way that other
-  origins are restricted. There are some additional restrictions as well, in order
-  to simplfy some complicated cases, and there are also some loosening of
-  same-origin restrctions in order to facilitate and ease adoption of suborigins
-  for developers.
+  origins are restricted. There are some additional restrictions as well, in
+  order to simplify some complicated cases.
 
-  ## Relationship with Sensitive Permissions ## {#sensitive-permissions}
+  ## Storage ## {#storage}
 
-  User agents MUST prevent a page running in a suborigin from accessing stateful
-  mechanisms (e.g., localStorage, sessionStorage, document.cookie) tied to the
-  parent, physical origin. Instead, the user agent SHOULD create a new object
-  tied to the namespaced suborigin.
+  The storage APIs, such as <a>localStorage</a> and <a>sessionStorage</a>, are
+  accessible from within suborigins. By nature of their APIs, they are bound to
+  the <a>origin</a> of the {{Document}}, which has the practical effect of
+  giving a separate {{Storage}} object to each <a>suborigin namespace</a>.
 
+  Issue: TODO: Should this storage section be dropped completely or at least
```

Comment:
One of the big problems today for deploying sandbox is that (iirc) Chrome for some unfathomable reason throws on accessing localStorage. Websites can't even override it by doing an Object.defineProperty on window. I really want the spec to say that this is non compliant at least for the case of suborigins.